lib/bootsnap/load_path_cache/cache.rb

```diff
@@ -40,11 +51,23 @@ def load_dir(dir)
         end
       end.freeze
 
+      def normalize_feature(feature)
```

Comment:
We should also conditionally define this method. I don't think MRI users should need to do all this calculation just to return false in `JRUBY_INTERNAL_LIBRARIES.key?(normalize_feature(feature))`

Reply:
I do now all the normalization while creating the hash. So in MRI case it is just an empty hash. So no conditional needed anymore or what do you think @rafaelfranca ?